internal/docstore/drivertest/drivertest.go

```diff
@@ -1067,10 +1075,8 @@ func filterHighScores(hs []*HighScore, f func(*HighScore) bool) []*HighScore {
 }
 
 // cleanUpTable delete all documents from this collection after test.
-func cleanUpTable(t *testing.T, coll *docstore.Collection) {
-	if err := coll.Query().Delete(context.Background()); err != nil {
-		t.Fatalf("%+v", err)
-	}
+func cleanUpTable(coll *docstore.Collection) error {
```

Comment:
You can put back the first argument as 
```
interface { Fatalf(string, ...interface{}) }
```

Reply:
Another idea I had is to put a call in with `withCollection` and `withTwoKeyCollection`, before we call the function. That would mean most of the tests wouldn't need to call it.

Reply:
Done.